app/partials/sign-in-form.cjsx

```diff
@@ -63,8 +63,8 @@ module.exports = createReactClass
             </a>
           </div>
 
-        else if @state.busy
-          <LoadingIndicator />
+        # else if @state.busy
```

Comment:
Should this be deleted?

Reply:
No, I don't think it should be commented out. I think this was leftover from when I was testing things while making the conversion. I'll put it back in. Good catch.